samples/view.json

```diff
@@ -41,7 +34,12 @@
     },
     "resource": {
       "count": 9
-    }
+    },
+    "in_foreground": true,
```

Comment:
I am a bit confused about where we stand on collecting `in_foreground` for views.
Do we collect them then?

Reply:
Oups I did not explicitly addressed that 😢 
I will remove them from the view to remove any possible confusion and only keep it in actions & errors.